spec/win32/unit/certstore_spec.rb

```diff
@@ -69,10 +69,46 @@
       end
     end
 
+    context "When deleting valid certificate" do
+      let (:store_name) { "ca" }
+      let (:certificate_name) { 'GeoTrust Global CA' }
+      before(:each) do
+        allow_any_instance_of(certbase).to receive(:CertDeleteCertificateFromStore).and_return(true)
+      end
+      it "return message of successful deletion" do
+        store = certstore.open(store_name)
+        delete_cert = store.delete(certificate_name)
+        expect(delete_cert).to eq("Deleted certificate GeoTrust Global CA successfully")
```

Comment:
`CertEnumCertificatesInStore` isn't stubbed so this spec may fail on a system which doesn't have a `GeoTrust Global CA` certificate `ca` store. Since appveyor isn't configured, this isn't very clear right now.
Anyway, if we decide to stop using `CertEnumCertificatesInStore` as per my previous comment, then the specs will change.